client/app/dbaas/logs/detail/roles/members/add/add-members.html

```diff
@@ -7,14 +7,15 @@
                data-on-dismiss="ctrl.cancel()"
                data-loading="ctrl.saving.loading">
         <oui-field data-label="{{::'logs_roles_members_modal_user' | translate}}">
-            <cloud-auto-complete
-                data-id="userName"
-                data-ng-model="ctrl.member.username"
+            <oui-select name="userName"
+                placeholder="logs-XX-XXXXX"
```

Comment:
Please refrain from using `placeholder`, instead write a text next to the input :)

Reply:
@FredericEspiau I'm not sure to understand your suggestion because the  `placeholder` attribute is not a deprecated one.

See: https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/select#deprecated